packages/tupaia-web/src/AppProviders.tsx

```diff
@@ -12,7 +12,7 @@ import { theme } from './theme';
 const queryClient = new QueryClient({
   defaultOptions: {
     queries: {
-      staleTime: 1000 * 60 * 5, // 5 minutes
+      staleTime: 1000 * 60 * 15, // 15 minutes
```

Comment:
Made the stale time longer as the data can take a long time to load and is not updated often and this came up in testing.